src/api/app/controllers/source_controller.rb

```diff
@@ -996,6 +996,9 @@ def package_command_release
       multibuild_container = params[:package].gsub(/^.*:/, '')
     end
 
+    # uniq timestring for all targets
+    time_string = Time.now.utc.strftime('%Y%m%d%H%M%S')
```

Comment:
You're only using this once, why do you put this in a variable?

Reply:
because it is used in a loop and must not change.

Reply:
I guess this gets a little more obvious if you put that assignment before that loop then.

Reply:
It's also only needed if you get into the `else` on line 1007.